app/assets/stylesheets/_subnav.scss

```diff
@@ -26,3 +26,86 @@
     }
   }
 }
+
+.study-subnav {
+  position: relative;
+  background-color: $color-off-white;
+  @include gradient-horizontal(
+    $start-color: $color-pale-green,
+    $end-color: $color-off-white,
+    $start-percent: 50%,
+    $end-percent: 50%
+  );
+  color: #aaa;
+
+  @media(max-width: $screen-xs-max){
+    display: none;
+  }
+
+  // Semi-transparent bottom border that sits on top of the
+  // coloured segments (saves giving each segment its own
+  // border, which would be tricky with the triangle ends).
+  &:after {
+    content: "";
+    display: block;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 1px;
+    background: rgba(0, 0, 0, 0.1);
+  }
+
+  ul {
+    @extend .list-unstyled;
+    @extend .clearfix;
+    margin: 0;
+  }
+
+  li {
+    float: left;
+    padding: 1em 2em 1em 1.5em;
+
+    color: #aaa;
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.